regression-tests.recursor-dnssec/recursortests.py

```diff
@@ -782,7 +783,9 @@ def tearDownAuth(cls):
 
     @classmethod
     def tearDownRecursor(cls):
-        cls.killProcess(cls._recursor)
+        p = cls.killProcess(cls._recursor)
+        if p.returncode != 0:
```

Comment:
Small nit here, isn't this a problem if we ever want to run the tests without any sanitizer enabled? Another untested option would be to set a specific `exitcode` in `LSAN_OPTIONS`.

Reply:
You're right. We also could bite the bullet and always change the exit code on TERM signals to 0. Have to think about this.

Reply:
Least intrusive way is to also accept -15 (killed by SIGTERM) for `p.returncode`